content/applications/services/helpdesk.rst

```diff
@@ -32,7 +31,6 @@ team is assigned to, select it from the :guilabel:`Company` drop-down menu.
    use only.
 
    .. image:: helpdesk/team-description-webform.png
-      :align: center
       :alt: View of a Helpdesk team's website form displaying the team description.
 
 Visibility & Assignment
```

Comment:
On line 87, I would reword slightly to: 

manually on each individual ticket, or through :guilabel:`Automatic Assignment`. Check the